account-kit/react/src/components/auth/card/error/connection-error.stories.tsx

```diff
@@ -6,7 +6,7 @@ const meta: Meta<typeof ConnectionError> = {
   component: ConnectionError,
   args: {
     connectionType: "passkey",
-    walletType: walletTypeConfig.CoinbaseWallet.key,
+    walletType: walletTypeConfig["com.coinbase.wallet"].key,
```

Comment:
Yeah not the biggest fan of this. Seems like it could get messy really fast. We should make a const of this or use my enum approach suggested below.

Reply:
Updated!